src/base/os.lua

```diff
@@ -504,7 +503,7 @@
 				return "chdir " .. path.translate(v)
 			end,
 			copy = function(v)
-				return "xcopy /Q /E /Y /I " .. path.translate(v)
+				return "xcopy /Q /E /Y /I " .. path.translate(v) .. " > nul"

```

Comment:
This is just to suppress confirmation message - i.e. **Files Copied: 1**
If there is going to be a error in a command arguments, xcopy will show the in console. I've tested that :)